src/routes/userRoute.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (app) => {
+  const userCtrl = require("../controllers/userCtrl");
+
+  app.post("/user/register", userCtrl.userRegister);
+  app.get("/users", userCtrl.getUsers);
+  app.get("/user/:_id", userCtrl.getOneUser);
+  app.delete("/user/:_id", userCtrl.deleteUser);
+  app.put("/user/:_id", userCtrl.updateUser);
```

Comment:
It might be appropriate to create the rights checking middleware now? Or was this planned in another ticket?

Reply:
in another ticket with role check